src/asm2wasm.h

```diff
@@ -664,6 +664,10 @@ void Asm2WasmBuilder::processAsm(Ref ast) {
     }, debug, false /* do not validate globally yet */);
   }
 
+  // if we see no function tables in the processing below, then the table still exists and has size 0
+
+  wasm.table.initial = wasm.table.max = 0;
```

Comment:
The constructor sets the max to `kMax` (indicating a growable table with no maximum). So the `initial` isn't strictly needed, just the `max`, but it feels nicer I think to set them both here, clarifying that we start the table at fixed size 0.